standard/expressions.md

```diff
@@ -594,8 +598,24 @@ The corresponding parameters for function member arguments are established as fo
 
 During the run-time processing of a function member invocation ([§12.6.6](expressions.md#1266-function-member-invocation)), the expressions or variable references of an argument list are evaluated in order, from left to right, as follows:
 
-- For a value parameter, the argument expression is evaluated and an implicit conversion ([§10.2](conversions.md#102-implicit-conversions)) to the corresponding parameter type is performed. The resulting value becomes the initial value of the value parameter in the function member invocation.
-- For a reference or output parameter, the variable reference is evaluated and the resulting storage location becomes the storage location represented by the parameter in the function member invocation. If the variable reference given as a reference or output parameter is an array element of a *reference_type*, a run-time check is performed to ensure that the element type of the array is identical to the type of the parameter. If this check fails, a `System.ArrayTypeMismatchException` is thrown.
+- For a value parameter, if the parameter’s passing mode is value
```

Comment:
>For a value parameter, if the parameter’s passing mode is value [](http://example.com/codeflow?start=2&length=63)

Does "value parameter" include parameters declared with an `in` modifier? Given that the next main bullet describes "input, output, or reference" parameters, it seems "value parameter" is a parameter without a modifier. In that case, isn't the "parameter's passing mode" always "value"?